.github/workflows/lint.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+name: Lint
```

Comment:
```suggestion

name: Lint
```

Reply:
:+1: applied that to https://github.com/cocotb/cocotb/pull/2962